CMakeLists.txt

```diff
@@ -1,23 +1,26 @@
-###########################################################################
-#
 # Copyright 2019 Qwant Research. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-# 
-###########################################################################
+
 cmake_minimum_required(VERSION 3.5)
 
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -lpthread -fpermissive ")
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_BUILD_TYPE RelWithDebInfo)
 
-include_directories(${PROJECT_SOURCE_DIR}/include  /usr/local/include/)
+include_directories(${PROJECT_SOURCE_DIR}/include/ /usr/local/include/)
 link_directories(/usr/local/lib/)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2 -std=c++11 -lpthread -fpermissive ")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -std=c++11 -lpthread -fpermissive ")
+
+set(CLASS_EXECUTABLE_API katanoisi_server)
```

Comment:
Change `katanoisi_server` to `katanoisi`

Reply:
Done